### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by cibersheep**
Should fix:

https://github.com/ubports/docs.ubports.com/issues/253

https://github.com/ubports/docs.ubports.com/issues/245

### CODE REVIEW

File: src/UbuntuToolkit/sortfiltermodel.cpp [hunk 1]
```diff
@@ -36,58 +36,65 @@ UT_NAMESPACE_BEGIN
...
- *             text: title
- *             subText: producer
- *         }
- *
- *         section.delegate: ListItem.Header { text: i18n.tr(section) }
- *         section.property: "title"
- *         section.criteria: ViewSection.FirstCharacter
- *     }
- * }
+ *        id: movies
```
Comment (mardy) [inline src/UbuntuToolkit/sortfiltermodel.cpp:47]: The indentation is wrong, please double check it (and the diff will become much smaller).

Comment (cibersheep) [reply src/UbuntuToolkit/sortfiltermodel.cpp:47]: Thanks. Modified
Tell me if that's ok now

**Comment by mardy** [comment]
Looks good to me, thanks! Just please squash the commits into one :-)

**Comment by cibersheep** [comment]
> Looks good to me, thanks! Just please squash the commits into one :-)

I may or may not know how to do that :)

Investigating...

**Comment by cibersheep** [comment]
Nope. I don't know how to do the squash. Could you give me a hand?

**Comment by mardy** [comment]
I think you did it, at least I can see only a single commit now. But just in case you still need it:

    git rebase -i xenial

Then in the text editor window which opens, change the initial `pick` into `f` in all but the first commit.

**Comment by cibersheep** [comment]
Oh, thanks. When I can clean my tree I'll push the changes again (properly I hope) :D

**Pull request closed without merging**